client/my-sites/invites/invite-form-header/index.jsx

```diff
@@ -12,7 +17,7 @@ export default React.createClass( {
 	},
 
 	getSiteName() {
-		return get( this.props, 'blog_details.title' );
+		return decodeEntities( get( this.props, 'blog_details.title', '' ) );
```

Comment:
Same situation as my previous comment, I think we should find a more generic place so we fix it globally.